libs/dmlf/src/colearn/muddle_learner_networker_impl.cpp

```diff
@@ -74,13 +74,14 @@ void MuddleLearnerNetworkerImpl::Setup(MuddlePtr mud, StorePtr update_store)
     serializers::MsgPackSerializer buf{payload};
 
     std::string                type_name;
+    std::string                algo_name;
     byte_array::ConstByteArray bytes;
     double                     proportion;
     double                     random_factor;
 
     auto source = std::string(fetch::byte_array::ToBase64(from));
 
-    buf >> type_name >> bytes >> proportion >> random_factor;
+    buf >> algo_name >> type_name >> bytes >> proportion >> random_factor;
 
     std::cout << "from:" << source << ", "
```

Comment:
Would it be useful to add algo_name and type_name to the cout message?

Reply:
Yes, I should do that.